_data-prepper/pipelines/configuration/sources/opensearch.md

```diff
@@ -55,7 +55,7 @@ opensearch-source-pipeline:
 
 ## Amazon OpenSearch Service
 
-The `opensearch` source can be configured for an Amazon OpenSearch Service domain by passing an `sts_role_arn` with access to the domain, as shown in the following example:
+The `opensearch` source can be configured for an Amazon OpenSearch Service domain by setting the `serverless` option to `true`, as shown in the following example:
```

Comment:
Similar comment as above. This section is about AOS, but then it tells how to configure for AOSS.